### DISCUSSION TITLE
Restrict Discord bot commands to specific Discord users, allow !status to public users

### DISCUSSION THREAD

**Discussion by asheroto** [seDirector/App#56] [Ideas]
Restrict Discord bot commands to specific Discord users, allow !status to public users

**Comment by DAYBR3AK1999** [comment]
Adding !status or /sed_status to @everyone and use a role such as seDirector Mod as a role which has full control of it.
People would just need to create that name and/or role on the server and give it to someone they trust so the bot can be used by them to restart servers or else.

**Reply by asheroto** [reply]
I've added this to the roadmap:

https://github.com/seDirector/App/projects/2#card-88148569
